docs/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps.md

```diff
@@ -135,7 +135,7 @@ The reference application (eShopOnContainers) tests were recently restructured a
 
 3. **Application functional/integration tests**, which focus on microservices integration, with test cases that exert several microservices. These tests are located in project **Application.FunctionalTests**.
 
-Unit and integration test per microservice are contained in a test folder in each microservice and Application a Load tests are contained under the test folder in the solution folder, as shown in Figure 6-25.
+Unit and integration test per microservice are contained in a test folder in each microservice, and Application and Load tests are contained under the test folder in the solution folder, as shown in Figure 6-25.
```

Comment:
Wouldn't that be better to end the sentence after the word "microservice" ? For e.g `....microservice. Application and Load tests ...`

Reply:
From what I’ve seen across the book, the comma + conjunction is used quite a lot, especially where ideas are linked; plus, I think that opting for a full stop will kind of break the flow of the sentence, though that could be fixed by adding a conjunction after the full stop.
Thanks @sughosneo

Reply:
What do you think of this @sughosneo ?
While unit and integration tests are contained in a test folder in each microservice, Application and Load tests are contained under the test folder in the solution folder, as shown in Figure 6-25.

Reply:
> What do you think of this @sughosneo ?
> While unit and integration tests are contained in a test folder in each microservice, Application and Load tests are contained under the test folder in the solution folder, as shown in Figure 6-25.

@zakaria-c , this looks good. One more final comment, we should change `Application` to "application" and `Load` to "load" ..

Reply:
@sughosneo , ok I've just updated it; thx!

Reply:
Looks good. Thanks @zakaria-c . We will merge the changes.